components/identifier-generator/front/src/feature/tabs/structure/edit/FamilyCodeEdit.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {PropertyEditFieldsProps} from '../PropertyEdit';
+import {AbbreviationType, FamilyCodeOperators, FamilyCodeProperty, Operator} from '../../../models';
+import {Field, NumberInput, SelectInput} from 'akeneo-design-system';
+import {useTranslate} from '@akeneo-pim-community/shared';
+import {OperatorSelector} from '../../../components';
+import styled from 'styled-components';
+
+const options = [
+  {value: AbbreviationType.TRUNCATE, label: 'pim_identifier_generator.structure.settings.code_format.type.first_chars'},
```

Comment:
```suggestion
  {value: AbbreviationType.TRUNCATE, label: 'pim_identifier_generator.structure.settings.code_format.type.truncate'},
```
Just to be consistent